src/js/media-chrome-button.js

```diff
@@ -157,9 +157,16 @@ class MediaChromeButton extends window.HTMLElement {
     const mediaControllerId = this.getAttribute(
       MediaStateReceiverAttributes.MEDIA_CONTROLLER
     );
+
     if (mediaControllerId) {
-      const mediaControllerEl = document.getElementById(mediaControllerId);
-      mediaControllerEl?.associateElement?.(this);
+      const rootNode = this.getRootNode();
```

Comment:
Trying to remember if there's any specific downsides to getRootNode. The TS parser at least doesn't like it when trying to use properties on it, so it needs to be ignored.

Reply:
we'll need to replace document.getElementById with this.getRootNode().getElementById for themes to work nicely with a decoupled media controller